### ISSUE TITLE
Issue with Step 6 of `9.4.1 LoadDocumentCallback`

### ISSUE THREAD

**Issue by multimeric** [w3c/json-ld-api#622]
[Current spec](https://www.w3.org/TR/json-ld11-api/#loaddocumentcallback)
[Source code](https://github.com/w3c/json-ld-api/blob/c05d6af1a1b13d273a63a79253e4608c8be929c1/index.html#L6405-L6428)

Assume the ideal case, where that the server has responded with `Content-Type: application/ld+json`:
* We skip step 4, since we have the `+json` suffix
* We skip step 5, since `application/ld+json` is specifically excluded
* Then, we reach step 6

Currently this is worded as:
> "Otherwise, the retrieved document's [Content-Type](https://tools.ietf.org/html/rfc2045#section-5) **is** neither application/json, application/ld+json, nor any other media type using a +json suffix as defined in [[RFC6839](https://www.w3.org/TR/json-ld11-api/#bib-rfc6839)]. Reject the promise passing a [loading document failed](https://www.w3.org/TR/json-ld11-api/#dom-jsonlderrorcode-loading-document-failed) error."

I think this is a mistake, and it should read as follows, because of course we do have `application/ld+json`:
> "Otherwise, **if** the retrieved document's [Content-Type](https://tools.ietf.org/html/rfc2045#section-5) is neither application/json, application/ld+json, nor any other media type using a +json suffix as defined in [[RFC6839](https://www.w3.org/TR/json-ld11-api/#bib-rfc6839)], reject the promise passing a [loading document failed](https://www.w3.org/TR/json-ld11-api/#dom-jsonlderrorcode-loading-document-failed) error."

This is almost certainly a mistake because step 7 proceeds with the algorithm, even though the current wording of step 6 implies that it will always fail here.

Happy to put in a PR here.

**Comment by TallTed** [comment]
To make the above clearer, the suggested change is just to add an `if` following the `Otherwise` at the start of the sentence.

**Comment by gkellogg** [comment]
Yes, I think that "Otherwise, if ..." is appropriate for step 6, as it is intended to by pass this step for JSON documents.
